Run.Script.R

```diff
@@ -0,0 +1,7 @@
+rm(list=ls())
```

Comment:
Please, exclude this file from the commit, because it is not related to the package.